app/src/main/res/xml/bci_settings_fragment.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <EditTextPreference
+        android:defaultValue="1"
+        android:title="Samples per second"
+        android:key="samples"
+        />
```

Comment:
Reformat the code using [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>], Keep the `/>` at the end of the last parameter instead on a new line.